src/wmtk/multimesh/MultiMeshSimplexVisitor.hpp

```diff
@@ -75,7 +75,8 @@ class MultiMeshSimplexVisitor
 
     template <typename MMVisitor_>
     friend class MultiMeshSimplexVisitorExecutor;
-    using Executor = MultiMeshSimplexVisitorExecutor<MultiMeshSimplexVisitor<cell_dimension, NodeFunctor>>;
+    using Executor =
+        MultiMeshSimplexVisitorExecutor<MultiMeshSimplexVisitor<cell_dimension, NodeFunctor>>;
```

Comment:
These changes in line break are a bit frightening. Someone is not using the correct formatting.